kroxylicious-systemtests/pom.xml

```diff
@@ -108,9 +108,9 @@
                         <phase>integration-test</phase>
                         <configuration>
                             <skipTests>${skipSTs}</skipTests>
-                            <excludes>
-                                <exclude>none</exclude>
-                            </excludes>
+<!--                            <excludes>-->
```

Comment:
Ditto

Reply:
Sorry forgot to remove them before performing the commit